src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php

```diff
@@ -19,6 +19,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('file', FileType::class, [
             'required' => true,
             'label' => 'kuma_translator.form.upload_file_choose',
+            'constraints' => array(
+                new NotBlank()
```

Comment:
And there is a `use Symfony\Component\Validator\Constraints\NotBlank;` use missing